### ISSUE TITLE
API/Command to play most recent entry of list

### ISSUE THREAD

**Issue by TASSDevon** [pietje666/plugin.video.vrt.nu#293]
I’d like it to play the latest News entry in the News category/directory list with a command, so I can instruct Home Assistant through its Kodi module to do this.

**Comment by mediaminister** [comment]
I made a pull request that implements a function that plays the latest episode of a tv show.
You can call this function through the [Kodi JSON-RPC API](https://kodi.wiki/?title=JSON-RPC_API)

For instance with curl:
```
curl -X POST -H "content-type:application/json" http://localhost:8080/jsonrpc -d '{"jsonrpc": "2.0", "method": "Player.Open", "params":{"item": {"file" : "plugin://plugin.video.vrt.nu/?action=playlatestepisode&tvshow=het-journaal" }}, "id" : "1"}'
```
I'm not familiar with Home Assistant but this should work with `media_player.kodi_call_method`

**Comment by TASSDevon** [comment]
Works, thanks!

Here's the HA setup:

<img width="601" alt="Screenshot 2019-06-08 at 15 06 02" src="https://user-images.githubusercontent.com/6917072/59147703-1dbb7100-89ff-11e9-9572-2c15bfb64b32.png">

**Comment by dagwieers** [comment]
Thanks for reporting back !

And thanks to @mediaminister for implementing it straight away ! :-)

**Comment by dagwieers** [comment]
I renamed the **Autostart** wiki section to **Automation** and added the information from @mediaminister to that wiki page. Improvements and feedback welcome.

**Comment by dagwieers** [comment]
@mediaminister Should we add a VRTNWS Live TV channel playing the latest episode in a loop ? That would be quite like the VRTNWS radio channel which is a continuous loop as well, or it is actually more like the VRTNWS radio stream channel which always starts from the beginning, but always the latest episode.

I think we should add it.

**Comment by mediaminister** [comment]
I don't see the added value of a "Het Journaal" loop, there are plenty of possibilities in the menus to watch the last episode of "Het Journaal".

But VRTNWS occasionally offers live events, breaking news, press conference reports or "Facebook Live" sessions on https://www.vrt.be/vrtnws/nl/
I think we should use a VRTNWS Live TV menu item in the first place for for these events. Maybe you can open a new issue for to further investigate and discuss this functionality.

**Comment by michaelarnauts** [comment]
I agree, I don't see the added value of a looped VRT NWS playlist.

**Comment by dagwieers** [comment]
It's the same value as the VRTNWS radio stream, with the added value that you also get any news flashes or news updates automatically. This is something that is often done on trade rooms to get the latest information when events unroll. So I don't think it is useful for the normal viewer that watches the latest episode only once, but rather for kiosks, trade floors, or places people queue

**Comment by DCL777** [comment]
Hallo,
I try to use this option but it's not working.  
Home Assistant changed the kodi call method to: kodi.call_method see screenshot:

![image](https://user-images.githubusercontent.com/55956951/65836961-2825b180-e2f3-11e9-876d-f96d773ea50f.png)

but this is not working at my setup.  So, i tried the basic command:
curl -X POST -H "content-type:application/json" http://localhost:8080/jsonrpc -d '{"jsonrpc": "2.0", "method": "Player.Open", "params":{"item": {"file" : "plugin://plugin.video.vrt.nu/?action=playlatestepisode&tvshow=het-journaal" }}, "id" : "1"}'

But it's also not working.  Kodi tried something (it's showing the same; some progress sign 3 or 4-times), but it's not playing.

Is something else changed?  Can someone confirm that it still should work like that?

Thanks
Dries

**Comment by DCL777** [comment]
This automation action reacts the same as the 2 methods above: 
![image](https://user-images.githubusercontent.com/55956951/65837155-74252600-e2f4-11e9-935f-a2ffd4ae1204.png)

**Comment by mediaminister** [comment]
This API was changed in version 2.0:
```
curl -X POST -H "content-type:application/json" http://localhost:8080/jsonrpc -d '{"jsonrpc": "2.0", "method": "Player.Open", "params":{"item": {"file" : "plugin://plugin.video.vrt.nu/play/latest/het-journaal" }}, "id" : "1"}'
```
More info in the wiki: https://github.com/pietje666/plugin.video.vrt.nu/wiki/Automation#playing-the-latest-episode-of-a-tv-show

**Comment by DCL777** [comment]
Yes, this is working.  
Thanks a lot,
Dries
